.projenrc.ts

```diff
@@ -80,6 +84,14 @@ const project = new typescript.TypeScriptProject({
   },
 });
 
+//
+// see https://nodejs.org/api/packages.html#exports
+project.addFields({
+  exports: {
+    './plugins': './lib/plugins/index.js',
+  },
```

Comment:
If that is your only export... maybe this could have been your `entrypoint` instead? Not that it makes a big difference (this is perhaps more future-proof)

Reply:
But I still want to actively prevent other imports, which only this can achieve. Right?

Reply:
Consider exporting `package.json` for consumers to determine which version of the CLI is being used.

Reply:
We add that if needed later on, not gonna bother with it just now.